.github/workflows/build.yaml

```diff
@@ -35,3 +38,58 @@ jobs:
           tags: "ghcr.io/${{ github.repository }}:${{ github.sha }}"
           cache-from: type=gha,scope=${{ github.repository }}
           cache-to: type=gha,scope=${{ github.repository }},mode=max
+
+  trivy-scan-code:
+    needs: build
+    name: Trivy Vulnerability Scan for Code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Aqua Security Trivy Scan
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          security-checks: 'vuln,secret,config'
+          ignore-unfixed: true
+          format: 'sarif'
+          output: 'trivy-results.sarif'
+
+      - name: Upload Trivy scan results to GitHub Security tab
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: 'trivy-results.sarif'
+          category: 'code'
+
+      - name: Image Scan ghcr
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'ghcr.io/${{ github.repository }}:${{ github.sha }}'
+          format: 'sarif'
+          output: 'trivy-image-results-ghcr.sarif'
+          exit-code: '1'
+          ignore-unfixed: true
```

Comment:
Can you remind me what `ignore-unfixed` do? And why are we using it?

Reply:
This flag excludes vulnerabilities for which there are no fixed versions. The idea is that if you can't do anything, you might as well not make a mistake

Reply:
Shouldn't we report them anyway? And leave the user to decide if the particular alert should be dismissed or to keep an eye on.

Reply:
Then check my comment on https://github.com/scality/runner-manager/pull/193#discussion_r1119367159

And review back this story

Reply:
Yeah, I think we can discuss it together on Google Meet, it will be easier for taking a decision